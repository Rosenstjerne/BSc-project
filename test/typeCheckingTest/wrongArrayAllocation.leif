var int[] A;

A = new int[true];