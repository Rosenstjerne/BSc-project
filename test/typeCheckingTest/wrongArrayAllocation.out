
Error in phase Type Checking, line 3:
    Type 'bool' can not be used to index into an array. Only allowed idexing type is 'int'
